app/assets/javascripts/discourse/app/lib/sidebar/categories-section/category-section-link.js

```diff
@@ -47,9 +46,25 @@ export default class CategorySectionLink {
   }
 
   get text() {
-    return htmlSafe(
-      categoryBadgeHTML(this.category, { link: false, categoryStyle: "bullet" })
-    );
+    return escapeExpression(this.category.name);
```

Comment:
Dropping `htmlSafe()` here means that Glimmer is gonna escape the value when rendering. If we `escapeExpression` manually, that means the value is going to be double-escaped. For example, if I check out this PR and name a category `Hobbies <blah>`, it looks like this in the sidebar:

<img width="269" alt="Screenshot 2022-08-15 at 15 12 38" src="https://user-images.githubusercontent.com/6270921/184651928-1a98820f-bfa0-46d2-815b-8d406d9809e7.png">

```suggestion
    return this.category.name;
```
(and then drop the `escapeExpression` import)

Reply:
All I was copy and pasting code from `helpers/category-link` without thinking about what it does. Have removed the `escapeExpression` function call.